contracts/apps/AragonApp.sol

```diff
@@ -31,6 +32,12 @@ contract AragonApp is AppStorage, Autopetrified, VaultRecoverable, ReentrancyGua
         _;
     }
 
+    modifier killSwitched {
```

Comment:
Wondering if we should directly bundle this into `auth()` or something similar as well. It may be tedious for developers to place this on every external, state-changing call (similar to `isInitialized()`).

We may also want to find a better name, e.g. `killSwitchProtected`.

Reply:
I think it may make sense but before coupling them we can keep working on other details and leave that decision for later. We will probably have a better understanding of how the whole kill-switch thing interacts with a DAO once we have finished some of the pending stuff.

OTOH, I like the new name you propose :)

Reply:
Yes, makes sense, although I wonder if, given the gas overhead, it could be useful to have that granularity to only apply the killSwitch to very critical functions.
Nevertheless, one could make the argue that if the function has an ACL protection is likely critical. :thinking: